e2e/test/scenarios/question/reproductions/38354-changing-source-database.cy.spec.js

```diff
@@ -0,0 +1,44 @@
+import {
+  getNotebookStep,
+  openNotebook,
+  popover,
+  restore,
+  visualize,
+} from "e2e/support/helpers";
+import { ORDERS_ID } from "metabase-types/api/mocks/presets";
+
+const QUESTION_DETAILS = {
+  query: {
+    "source-table": ORDERS_ID,
+    limit: 5,
+  },
+};
+
+describe("issue 38354", () => {
+  beforeEach(() => {
+    restore("postgres-12");
```

Comment:
curious, why postgres-12?

Reply:
Because we need another database, and the snapshot already exists.
Did you mean - why not another H2?